sample_install-deps.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+## Modify the following lines to install dependencies for your project.
+echo "Installing dependencies for non-ROS packages"
+apt update -q
+apt install -y --no-install-recommends git libgoogle-glog-dev \
+    libgflags-dev \
+    libatlas-base-dev \
+    libeigen3-dev \
+    libsuitesparse-dev
+
+mkdir -p ./deps && cd ./deps
```

Comment:
really like the creation of the .deps here. Consider removing the .deps folder instead of the ceres-solver here

Reply:
I meant to remove the deps folder, this was an error on my part.